TheWarWithin/EvokerPreservation.lua

```diff
@@ -116,19 +129,19 @@ spec:RegisterTalents( {
     punctuality                     = {  93260, 371270, 1 }, -- Reversion has 2 charges.
     renewing_breath                 = {  93268, 371257, 2 }, -- Dream Breath healing is increased by 15%.
     resonating_sphere               = {  93258, 376236, 1 }, -- Temporal Anomaly applies Echo at 30% effectiveness to the first 5 allies it passes through.
-    reversion                       = {  93338, 366155, 1 }, -- Reverse an ally's injuries, instantly healing them for 15% of damage taken in the last 5 sec and an additional 30,334 over 12 sec. When Reversion critically heals, its duration is extended by 2 sec, up to a maximum of 12 sec.
-    rewind                          = {  93337, 363534, 1 }, -- Rewind 33% of damage taken in the last 5 seconds by all allies within 40 yds. Always heals for at least 20,994. Healing increased by 100% when not in a raid.
+    reversion                       = {  93338, 366155, 1 }, -- Reverse an ally's injuries, instantly healing them for 15% of damage taken in the last 5 sec and an additional 58,287 over 15.6 sec. When Reversion critically heals, its duration is extended by 2 sec, up to a maximum of 12 sec.
```

Comment:
Just a quick note; it's good to be naked when using the skeleton generator, so that the durations aren't impacted by haste on gear.

I don't know if that's the case here, or if Reversion naturally lasts 15.6 seconds.  Not a bug, no fix needed -- just info.